src/parsers/TAMU_data.jl

```diff
@@ -0,0 +1,108 @@
+
+"""
+    TAMU_System(TAMU_FOLDER::AbstractString;
+                     kwargs...)
+
+Creates a system from a PSS/e .RAW (v33) load flow case, and an associated .csv with MW load
+time series data. The format is established by the
+[Texas A&M University Test Case Archive](https://electricgrids.engr.tamu.edu/electric-grid-test-cases/)
+
+The general format for data is
+folder:
+   [casename].raw
+   [casename]_load_time_series_MW.csv
+
+# Arguments
+- `directory::AbstractString`: directory containing RAW and CSV files
+
+# Examples
+```julia
+sys = parse_TAMU(
+    "./ACTIVSg25k",
+    configpath = "ACTIVSg25k_validation.json",
+    bus_name_formatter = x->string(x["name"]*"-"*string(x["index"])),
+    load_name_formatter = x->strip(join(x["source_id"], "_"))
+)
+```
+"""
+
+function TAMU_System(TAMU_FOLDER::AbstractString; kwargs...)
```

Comment:
Even though it's a proper noun, I suggest that it's better to stick with naming conventions.  It's always predictable.  I would use `parse_tamu`, `TamuSystem`, `tamu_folder`.  At least `TAMUSystem`.

Reply:
I went with `TamuSystem` and `tamu_folder`.